src/Surfnet/StepupGateway/GatewayBundle/Controller/GatewayController.php

```diff
@@ -134,6 +134,22 @@ public function consumeAssertionAction(Request $request)
         }
 
         $adaptedAssertion = new AssertionAdapter($assertion);
+
+        if (!$adaptedAssertion->hasSubject()) {
+            $logger->critical('Received Response without eduPersonTargetedID (EPTI)');
+            return $this->render('unrecoverableError', [
+                'message' => 'The "urn:mace:dir:attribute-def:eduPersonTargetedID" is not present'
```

Comment:
Same suggestion as above: The "urn:mace:dir:attribute-def:eduPersonTargetedID" **attribute** is not present